pkg/katautils/oci.go

```diff
@@ -11,6 +11,9 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+
+	"github.com/containers/libpod/pkg/rootless"
```

Comment:
@gabibeyer I dont see the vendoring for this. Are you planning to add the vendoring for this? 
In any case, I see that you are using just a couple of functions from the package. It is better to copy just the required function if possible in our own repo, rather than vendor the entire package.

Reply:
For reference, see:

- https://github.com/kata-containers/community/blob/master/VENDORING.md#vendor-a-newly-imported-package
- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#subsystem